howto/invoke-and-discover-services/README.md

```diff
@@ -96,6 +96,16 @@ curl http://localhost:3500/v1.0/invoke/cart/method/add -X DELETE
 
 Dapr puts any payload return by their called service in the HTTP response's body.
 
+### Namespaces
+
+When running on [namespace supported platforms](../../reference/api/service_invocation_api.md#namespace-supported-platforms), you can add the namespace of your choice to the app ID:
```

Comment:
nit, how about active tone: You include namespace with your ID when running on a supporting platform?

Reply:
Changed.